examples/flow/flow_pois.py

```diff
@@ -58,6 +61,33 @@ def get_set(file_name):
 wrapper = LammpsWrapper()
 wrapper.CM[CUBA.NUMBEROF_TIME_STEPS] = 10000
 wrapper.CM[CUBA.TIME_STEP] = 0.003
+wrapper.CM[CUBA.PAIR_POTENTIALS] = ("lj:\n"
```

Comment:
I have a few questions about PAIR_POTENTIALS

- Is `PAIR_POTENTIALS` only valid in a configuration data container?
- Does the user need to edit the contents at runtime (i.e. can we always point into a file to load the pair potentials configuration)?
- Could `PAIR_POTENTIALS` be a list of PairPotentials objects? This means that one can create a PAIR_POTENTIALS setup programmatically. There could be also a method that reads a pair_potentials.yml file and returns a list of PairPotential instaces